README.md

```diff
@@ -19,11 +19,11 @@ If you have an iPhone and prefer to use an app instead of the website, Aspine is
 
 ![CRLS Clock Image](images/crls-clock.png)
 
-* Grade Comparison via boxplot
+* Grade Comparison via Box Plot
 
 ![Example Boxplot Image](images/boxplot.png)
 
-* Export and import your grades
+* Export and Import your Grades
```

Comment:
It is good to normalize the letter case here, but I think it would be better to normalize to sentence case instead of title case. Your thoughts @jadebuckwalter?

Reply:
@psvenk Yes, that makes sense, but it also contributes to greater inconsistencies, like "Aesthetic Grade Checking" being in title case while "Export and import your grades" being in sentence case. If that's ok, then I'm all for switching to sentence case.

Reply:
@jadebuckwalter I meant to suggest putting the entire section in sentence case. The reason why I only commented on these lines is that GitHub did not allow me to select lines that you left untouched.

Reply:
@psvenk I see. Sorry for the confusion.